src/NuGet.Core/NuGet.Build.Tasks/RestoreSettingsUtils.cs

```diff
@@ -51,7 +58,8 @@ public static ISettings ReadSettings(string solutionDirectory, string restoreDir
                 return Configuration.Settings.LoadDefaultSettings(
                     directory,
                     configFileName,
-                    null);
+                    null,
```

Comment:
nit: named param :) I know it's old code, but it'd improve readability.

Reply:
Not a nit, I think all params that are `null`, `true`, `false`, or any other literal should have a name.